src/schemes/fluid/viscosity.jl

```diff
@@ -85,31 +85,77 @@ struct ArtificialViscosityMonaghan{ELTYPE}
     end
 end
 
-@inline function (viscosity::ArtificialViscosityMonaghan)(particle_system, neighbor_system,
-                                                          v_particle_system,
-                                                          v_neighbor_system,
-                                                          particle, neighbor, pos_diff,
-                                                          distance, sound_speed, m_a, m_b,
-                                                          rho_mean)
+@doc raw"""
+    ViscosityMorris(; nu, epsilon=0.01)
+
+Viscosity by Moris (Moris et al. 1997).
+```math
+f_{ab} = \sum_w \frac{m_b(eta_a+eta_b)}{||r_{ab}||^2+(\epsilon h_{ab})^2}  \nabla W_{ab} \cdot r_{ab}\cdot v_{ab},
+```
```

Comment:
````suggestion

To the force ``f_{ab}`` between two particles ``a`` and ``b`` due to pressure gradients, an additional force term ``\tilde{f}_{ab}`` is added with
```math
\tilde{f}_{ab} = m_a m_b \frac{(\eta_a + \eta_b) r_{ab} \cdot \nabla W_{ab}}{\rho_a \rho_b (\Vert r_{ab} \Vert^2 + \epsilon h^2)} v_{ab},
```
````

Reply:
Equation not resolved.